pkgs/common-updater/scripts/list-archive-versions

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# lists all available versions listed for a package in a site (http)
+
+archive=""  # archive url
```

Comment:
It looks like it is more of a directory listing URL, rather than archive URL.

Reply:
I agree. I am going to change the script name and the variable name.